pkg/apiclient/apiclient.go

```diff
@@ -64,13 +64,13 @@ const (
 	EnvArgoCDServer = "ARGOCD_SERVER"
 	// EnvArgoCDAuthToken is the environment variable to look for an Argo CD auth token
 	EnvArgoCDAuthToken = "ARGOCD_AUTH_TOKEN"
-	// EnvArgoCDgRPCMaxSizeMB is the environment variable to look for a max gRPC message size
-	EnvArgoCDgRPCMaxSizeMB = "ARGOCD_GRPC_MAX_SIZE_MB"
 )
 
 var (
 	// MaxGRPCMessageSize contains max grpc message size
-	MaxGRPCMessageSize = env.ParseNumFromEnv(EnvArgoCDgRPCMaxSizeMB, 200, 0, math.MaxInt32) * 1024 * 1024
+	MaxGRPCMessageSize = env.ParseNumFromEnv(common.EnvArgoCDgRPCMaxSizeMB, 200, 0, math.MaxInt32) * 1024 * 1024
+	// MaxGRPCRetriesNum contains max grpc retries
+	MaxGRPCRetriesNum = env.ParseNumFromEnv(common.EnvArgoCDgRPCMaxRetries, 3, 0, math.MaxInt32)
```

Comment:
I'd also love to see these as command params instead of global settings. But since the pattern is already established, I'm less opinionated about this one than the cmp server client. Maybe take a stab at converting it to a command flag? If it's too gnarly, we can just leave it.

Reply:
Yes, many functions create grpc connections, and if using flag causes a lot of repetitive field passing, perhaps parsing from environment variables would be a better approach?

Reply:
We have a concept of "global CLI flags" that are made available for all commands. This would probably be a good candidate for including among those global flags.